ServerlessLibraryAPI/wwwroot/items.json

```diff
@@ -7,7 +7,8 @@
     "language": "csharp",
     "type": "functionapp",
     "authortype": "Community",
-    "runtimeversion": "v1"
+    "runtimeversion": "v1",
+    "tags": [ "functionapp", "csharp", "OpenALPR" ]
```

Comment:
For the other samples, i think we should copy the tags from the Solution Area section in this list 
https://github.com/Azure/ServerlessLibrary/issues?utf8=%E2%9C%93&q=+label%3A%22add+to+library%22+  .

Thoughts ?

Reply:
Right. That can be done later